packages/core/src/initialization/make-index-api.ts

```diff
@@ -26,6 +26,17 @@ export function makeIndexApi(
   }
 
   const indexApi = buildIndexing(indexingConfig, logger, network)
+  // TODO(CDB-2078): replace env var with config option from ceramic_config
+  if (
+    process.env.CERAMIC_ENABLE_EXPERIMENTAL_SYNC === 'true' &&
```

Comment:
I thought we were getting rid of this env var?  Why is this even still required?  Isn't HDS on by default now?

Reply:
not yet on by default until qualification has been run on main-net. Bit concerned around Infura API calls as well atm with current non-optimization. If necessary I can remove it as part of a final RC build if necessary.